features/admin.applications.v1/components/settings/attribute-management/advance-attribute-settings.tsx

```diff
@@ -422,17 +446,29 @@ export const AdvanceAttributeSettings: FunctionComponent<AdvanceAttributeSetting
                             </Hint>
                         </>
                     ) }
+                    { !resolveSubjectAttributeHiddenStatus () && isEmpty(dropDownOptions) && (
+                        <Message info>
+                            <Icon name="info circle" />
+                            { t("applications:forms.advancedAttributeSettings" +
+                        ".sections.subject.fields.subjectAttribute.info") }
+                        </Message>
+                    ) }
+
                     <Field.Dropdown
                         ariaLabel="Subject attribute"
                         name="subjectAttribute"
                         label={
                             t("applications:forms.advancedAttributeSettings" +
                                 ".sections.subject.fields.subjectAttribute.label")
                         }
+                        placeholder = {
+                            t("applications:forms.advancedAttributeSettings" +
+                                ".sections.subject.fields.subjectAttribute.placeholder")
+                        }
                         required={ claimMappingOn }
                         value={ selectedSubjectValue }
                         options={ dropDownOptions }
-                        hidden={ resolveSubjectAttributeHiddenStatus() }
+                        hidden={ resolveSubjectAttributeHiddenStatus() || isEmpty(dropDownOptions) }
```

Comment:
shall we get this to a separate function for readability ?

Reply:
check other places as well

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/5879/commits/e73261ed2dbc028fa40283f153f2e178df3b5e85